content/version/1/4/code-of-conduct.hu.md

```diff
@@ -0,0 +1,54 @@
++++
+version = "1.4"
+aliases = ["/version/1/4"]
++++
+
+# Közreműködők Magatartási Kódexe
+
+## Vállalásunk
+
+A nyitott és barátságos környezet megteremtése érdekében mi, mint résztvevők és fenntartók vállaljuk azt, hogy a projektünk és közösségünk zaklatásmentes mindenki számára, függetlenül az életkortól, test mérettől, fogyatékosságtól, etnikai hovatartozástól, nemi jellemzőktől, nemi identitástól és kifejezésétől, a tapasztalattól, az oktatástól, a társadalmi-gazdasági helyzettől, az állampolgárságtól, a személyes
+megjelenéstől, fajtól, vallástól vagy szexuális identitástól és orientációtól.
+
+## Szabályaink
+
+Viselkedési példák, amelyek segítik a pozitív környezet létrehozását:
+
+* Barátságos és befogadó nyelv használata
+* Különböző szempontok és tapasztalatok tiszteletben tartása
+* Tisztelettudóan elfogadja a konstruktív kritikát
+* A közösség érdekének szem előtt tartása
+* Más közösségi tagok felé mutatott empátia
+
+Példák az elfogadhatatlan viselkedésre a résztvevők részéről:
+
+* A szexuális nyelvezet vagy ilyen képek használata, valamint nemkívánatos szexuális figyelem felkeltés vagy érdeklődés
+* Trollkodás, sértő vagy lekicsinylő megjegyzések, valamint személyes vagy politikai támadások
+* Nyilvánosság előtti vagy személyes zaklatás
+* Mások személyes adatainak, mint például fizikai vagy elektronikus címének közzététele, az érintett kifejezett engedélye nélkül
+* Egyéb magatartás, amelyek átlépik a szakma ésszerű határait
+
+## Felelősségünk
+
+A projekt fenntartói felelősek az elfogadható viselkedési szabályok tisztázásáért, és elvárható tőlük, hogy megtegyék a megfelelő, és tisztességes korrekciós intézkedéseket az elfogadhatatlan viselkedések esetén.
+
+A projektgazdáknak joguk és felelősségük, hogy eltávolítsák, szerkesszék, vagy elutasítsák a megjegyzéseket, kódokat, wiki szerkesztéseket, hibákat, észrevételeket, és egyéb hozzájárulásokat, amelyek nem illeszkednek e Magatartási Kódexhez, vagy ideiglenesen, vagy véglegesen kitiltják a tartósan fenyegető, sértő vagy káros magatartást tanúsító résztvevőt.
+
+## Hatály
+
+Ez a Magatartási Kódex minden projektterületen érvényes, és akkor is érvényes, amikor az egyén nyilvános helyeken képviseli a projektet vagy annak közösségét. Például ilyen események a projekt vagy a közösség képviseletére használt hivatalos e-mail cím használata, vagy hivatalos szociális médiafiókon keresztül történő közzétételek, vagy kijelölt képviselőként egy online vagy offline eseményen történő részvétel. A hivatalos képviseletét a projektnek a projekt fenntartói még részletesebben definiálhatják és tisztázhatják.
```

Comment:
Eliminate the redundant usage of "vagy" a comma separated list should suffice

Reply:
Ok, thanks!